uploadNeuroDB/minc_insertion.pl

```diff
@@ -380,13 +387,15 @@
 ################################################################
 ## Get acquisition protocol (identify the volume) ##############
 ################################################################
-my ($acquisitionProtocol,$acquisitionProtocolID,@checks)
-  = $utility->getAcquisitionProtocol(
+if(!defined($acquisitionProtocol)) {
+  my ($acquisitionProtocol,$acquisitionProtocolID,@checks)
```

Comment:
@gluneau,

$acquisitionProtocol is no longer accessible for the next check $utility->registerScanIntoDB() (line 416)